Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.io/alpine:latest
+
+RUN apk add --no-cache git && \
+    mkdir /workdir
+
+COPY git-chglog /usr/local/bin/git-chglog
+
+WORKDIR /workdir
+RUN chmod +x /usr/local/bin/git-chglog
```

Comment:
Technically speaking this line is not needed.

Reply:
I tried without the 'chmod +x' and it didn't work.

Reply:
If you let `goreleaser` build the image, the binary will be executable, but we can be on a safe side and leave it as is.